workshop/controllers/ui/components/themePreviewer.lua

```diff
@@ -29,16 +136,16 @@ for _,prop in pairs(themer.types) do
       fontFile = "local:OpenSans-SemiBold.ttf"
    }, "backgroundText")
 
-   local count = 0
+   count = 0
    for _,v in pairs(theme[prop]) do if type(v) == "colour" then count = count + 1 end end
 
-   local size = 1/count
-   local i = 0
+   size = 1/count
+   i = 0
 
    for k,v in pairs(theme[prop]) do
       if type(v) == "colour" then
-         local ch,cs,cv = v:getHSV()
-         local btn = ui.create("guiTextBox", themeProperty, {
+         ch,cs,cv = v:getHSV()
```

Comment:
why is this done?

Reply:
if you look its still there, its just not a local varible anymore and that is triping it up

Reply:
everything still works

Reply:
I mean, I don't see these three variables being used anywhere?

Reply:
i was having issues with local and non local varibles, I could not find the ones causing problems, so I set them all to global.